exodus-installer.sh

```diff
@@ -52,19 +57,47 @@ exodus_download() {
 }
 
 
+# Download and check the exodus package to verify
+# SHA hash
+exodus_verify_hashes() {
+  #
+  # If JP's key doesn't exist...
+  if ! gpg --list-public-keys --with-colons --fixed-list-mode --with-fingerprint | grep -q '^fpr:::::::::12408650E2192FEBE4E7024C9D959455325B781A:$'
+  then
+    jpKey='https://keybase.io/jprichardson/pgp_keys.asc?fingerprint=12408650e2192febe4e7024c9d959455325b781a'
+    # ...Import JP Richardson's Public Key
+    curl -s $jpKey | gpg --import -q
```

Comment:
We should check exit code for gpg import.

Reply:
GPG import status is checked, return 1 on failure.